manifests/instance.pp

```diff
@@ -0,0 +1,55 @@
+# @summary Configure an Anubis instance
+#
+# @param target
+#   The target Anubis should forward to
+#
+# @param settings
+#   A hash of settings to be set in the environment file
+#
+# @param ensure
+#   What state of the instance should be ensured, present or absent.
+#
+# @param bot_policies
+#   Content of the botPolicies file, if any.
+#   If undef (the default) the default policies file is used.
+
+define anubis::instance (
+  Stdlib::HTTPUrl $target,
+  Hash[String, String] $settings = {},
+  Enum['present', 'absent'] $ensure = 'present',
+  Optional[String[1]] $bot_policies = undef,
+) {
+  include anubis
```

Comment:
I think the package should be installed before you create the files and this is probably the easiest fix
```suggestion
  require anubis
```